src/Sylius/Bundle/AdminBundle/Twig/Component/Product/GenerateVariantsFormComponent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Twig\Component\Product;
+
+use Sylius\Bundle\AdminBundle\TwigComponent\HookableComponentTrait;
+use Sylius\Component\Core\Model\Product;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+use Symfony\UX\LiveComponent\LiveCollectionTrait;
+
+#[AsLiveComponent(name: 'sylius_admin:product:generate_variants_form', template: '@SyliusAdmin/Product/generate_variants/form.html.twig')]
+final class GenerateVariantsFormComponent
```

Comment:
I'm uncertain about including verbs in the class name. Nevertheless, we do not have any specific convention regarding this.

```suggestion
final class ProductVariantsGeneratorFormComponent
```
or
```suggestion
final class VariantsGeneratorFormComponent
```